driver/normalizer/normalizer.go

```diff
@@ -174,35 +222,40 @@ var Normalizers = []Mapping{
 		}),
 		role.Name),
 
-	MapSemantic("Attribute", uast.Identifier{}, MapObj(
-		Obj{"attr": Var("name")},
-		Obj{"Name": Var("name")},
-	)),
-
-	MapSemantic("arg", uast.Argument{}, MapObj(
-		Obj{
-			uast.KeyToken: Var("name"),
-			"default":     Var("init"),
-		},
-		Obj{
-			"Name": identifierWithPos("name"),
-			"Init": Var("init"),
-		},
-	)),
-
 	MapSemantic("arg", uast.Argument{}, MapObj(
-		Obj{
-			uast.KeyToken: Var("name"),
+		Fields{
+			{Name: uast.KeyToken, Op: Var("name")},
+			{Name: "default", Optional: "opt_def", Op: Var("init")},
+			// No problem dropping this one, it's used by an internal interpreter optimization/cache
+			// without semantic meaning
+			{Name: "ctx", Optional: "opt_ctx", Op: Any()},
+			// TODO: change this once we've a way to store other nodes on semantic objects
```

Comment:
```suggestion
			// FIXME: change this once we've a way to store other nodes on semantic objects
```
a bit more comments like this below

Reply:
Done (all) in next commit.